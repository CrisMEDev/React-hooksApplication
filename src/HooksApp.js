import React from 'react'

const HooksApp = () => {
    return (
        <>
            <h1>Hola mundo, Cristian</h1>
        </>
    );
}

export default HooksApp;
