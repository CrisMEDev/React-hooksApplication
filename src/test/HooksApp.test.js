import { shallow } from "enzyme";

import HooksApp from "../HooksApp";

describe('Pruebas en en componente <HooksApp />', () => {

    test('Debe mostrar el componente correctamente', () => {

        const wrapper = shallow( <HooksApp /> );

        expect( wrapper ).toMatchSnapshot();
        
    });
    

});
