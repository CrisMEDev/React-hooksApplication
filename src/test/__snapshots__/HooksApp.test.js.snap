// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en en componente <HooksApp /> Debe mostrar el componente correctamente 1`] = `
<Fragment>
  <h1>
    Hola mundo, Cristian
  </h1>
</Fragment>
`;
